app/views/scheduled_tests/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Notes:</strong>
+  <%= @scheduled_test.notes %>
+</p>
+
+<p>
+  <strong>Collection:</strong>
+  <%= @scheduled_test.collection %>
+</p>
+
+<p>
+  <strong>Test ids:</strong>
+  <%= @scheduled_test.test_ids %>
+</p>
+
+<p>
+  <strong>Next test:</strong>
+  <%= @scheduled_test.next_test %>
+</p>
+
+<p>
+  <strong>Recurring:</strong>
+  <%= @scheduled_test.recurring %>
+</p>
```

Comment:
Need to indicate the units of recurrence. Assuming this is days currently.

Also need to apply some level of validation as it is possible to provide a negative value here.